Dockerfile

```diff
@@ -1,13 +1,21 @@
-FROM node:9.2.0-alpine
+FROM node:9-slim
 
-ENV SCREENIE_VERSION=1.2.0
+ENV SCREENIE_VERSION=2.0.0
+ENV SCREENIE_CHROMIUM_ARGS="--no-sandbox"
```

Comment:
We shouldn't let this be insecure by default, I'd say.

Reply:
Well I didnt get it to run in sandbox mode in Docker. I followed https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker also, but the run command that the example has includes `cap_add` which Im not sure can be included in a docker stack file.

`Failed to move to new namespace: PID namespaces supported, Network namespace supported, but failed: errno = Operation not permitted`

Reply:
OK, I'm fine with this for now then, since this is what we'll use anyhow. I suggest adding a note to the readme though, that running it with Chrome sandboxing isn't supported.